app.js

```diff
@@ -48,6 +52,8 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(fileUpload());
 
+var token = 'SECRET_TOKEN_f8ed84e8f41e4146403dd4a6bbcea5e418d23a9';
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-high.svg" width="20"/></sub>&nbsp; Hardcoded Secret

Avoid hardcoding values that are meant to be secret. Found a hardcoded string used in here.

<h6>
<a href="https://cwe.mitre.org/data/definitions/547.html">CWE-547</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 812</a> | Line 55</h6>